### DISCUSSION TITLE
QR Code Autofocus Problems - iOS 16.3 Safari/Chrome

### DISCUSSION THREAD

**Discussion by akula34** [Budibase/budibase#9483] [Help]
The QR Code scanner appears to use the proper autofocus on my computer for the webcam, 16" 2022 M1 MBP, but it does **not** use autofocus of the iOS version.

As a result, I'm not able to scan any barcodes or QR codes using my mobile device (iPhone 14 Pro Max), since the camera will not focus on near objects. 

Do I need to raise an issue?

Anyone else having similar problems? I can't be the only one.

**Comment by melohagan** [comment]
Hey @akula34 

If you are able to provide reproducible steps then we would appreciate if you could raise a bug issue.

**Comment by DennisNissenUX** [comment]
Hey, I have the same issue.
To reproduce just get yourself a small qr code and try to scan it with the budibase qr code scanner. the camera image will be too blurry to scan the code. The reason seems to be, that the camera can't refocus on the small image.
